lib/app/modules/home/views/home_view.dart

```diff
@@ -328,262 +328,249 @@ class HomeView extends GetView<HomeController> {
-                                                            MainAxisAlignment
-                                                                .start,
-                                                        children: [
-                                                          Expanded(
-                                                            flex: 3,
-                                                            child: Column(
-                                                                mainAxisAlignment:
-                                                                    MainAxisAlignment
-                                                                        .center,
-                                                                crossAxisAlignment:
-                                                                    CrossAxisAlignment
-                                                                        .start,
-                                                                children: [
-                                                                  Text(
-                                                                      repeatDays.replaceAll(
-                                                                          "Never",
-                                                                          "One Time"),
-                                                                      style: Theme.of(
-                                                                              context)
-                                                                          .textTheme
-                                                                          .bodySmall!
-                                                                          .copyWith(
-                                                                              fontWeight: FontWeight.w500,
-                                                                              color: (alarm.isEnabled == true) ? kprimaryColor : kprimaryDisabledTextColor)),
-                                                                  Row(
-                                                                    children: [
-                                                                      Text(
-                                                                        time12[
-                                                                            0],
+                                                          BorderRadius.circular(
+                                                              18),
+                                                    ),
+                                                    child: Center(
+                                                      child: Padding(
+                                                        padding:
+                                                            const EdgeInsets
```

Comment:
Currently, it looks a bit too clustered when the enabled feature icons show up, maybe we should add some top and bottom padding when any of the features is enabled.

You can take advantage of the isChallengeEnabled and isAutoDismissalEnabled functions in Utils class